core/src/main/java/org/mobilitydata/gtfsvalidator/model/SampleNotice.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.internal.LinkedTreeMap;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+
+/**
```

Comment:
FWIW, these types of model classes could be represented in far fewer lines of code using Kotlin rather than Java. 

It would be something like:

```kotlin
data class SampleNotice(
    val code: String, 
    val severity: SeverityLevel, 
    val totalNotices: Int, 
    val sampleNotices: List<LinkedTreeMap<String, Object>>
  ) {

  fun isError(): Boolean {
    return severity.ordinal() >= SeverityLevel.ERROR.ordinal()
  }
}
```

Possible future improvement if you don't want to tackle that in this PR.